tools/packages.config

```diff
@@ -1,4 +1,4 @@
 <?xml version="1.0" encoding="utf-8"?>
 <packages>
-	<package id="Cake" version="0.18.0-alpha0123" />
+	<package id="Cake" version="0.18.0-alpha0033" />
```

Comment:
@daveaglick do you know, roughly, when you are planning on shipping the latest Cake.Wyam addin?  It is the last one that is being used by Cake.Recipe that hasn't been updated to use 0.17.0 yet.  @pascalberger sent a PR through to update it recently.  Just wondering when we can expect that to ship.  Thanks!